1.2.0 fails to compile with latest dolphin (or kcoreaddons?)

[build.log](https://paste.2x.ax/~saiko/6ff98f076e1199c5226c577e3bc14eb33a166be3)

Same here with Dolphin 20.08.1 (KDE Framework 5.73.0, Qt 5.15.0) under Manjaro KDE

```
cd plugin/ && ( test -e Makefile || /usr/bin/qmake -o Makefile /var/tmp/pamac-build/dolphin-seafile-plugin/dolphin-seafile-plugin-1.2.0/plugin/plugin.pro ) && make -f Makefile 
make[1]: Verzeichnis „/var/tmp/pamac-build/dolphin-seafile-plugin/build/plugin“ wird betreten
g++ -c -pipe -O2 -pthread -Wall -Wextra -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT_ASCII_CAST_WARNINGS -DQT_NO_DEBUG -DQT_PLUGIN -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I../../dolphin-seafile-plugin-1.2.0/plugin -I. -isystem /usr/include/Dolphin -isystem /usr/include/searpc -isystem /usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/qt -isystem /usr/include/qt/QtWidgets -isystem /usr/include/qt/QtGui -isystem /usr/include/KF5/KIOCore -isystem /usr/include/KF5/KCoreAddons -isystem /usr/include/KF5/KService -isystem /usr/include/KF5/KConfigCore -isystem /usr/include/qt/QtCore -I. -I/usr/lib/qt/mkspecs/linux-g++ -o fileviewseafileplugin.o ../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.cpp
In Datei, eingebunden von ../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.cpp:1:
../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.h:20:19: Fehler: »QList<QAction*> FileViewSeafilePlugin::actions(const KFileItemList&) const« als »override« markiert, überschreibt aber nicht
   20 |  QList<QAction *> actions(const KFileItemList &items) const override;
      |                   ^~~~~~~
In Datei, eingebunden von /usr/include/KF5/KCoreAddons/KPluginFactory:1,
                 von ../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.h:5,
                 von ../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.cpp:1:
/usr/include/KF5/KCoreAddons/kpluginfactory.h: In Instanziierung von »static QObject* KPluginFactory::createInstance(QWidget*, QObject*, const QVariantList&) [with impl = FileViewSeafilePlugin; ParentType = QObject; QVariantList = QList<QVariant>]«:
/usr/include/KF5/KCoreAddons/kpluginfactory.h:435:21:   erfordert durch »QObject* (* KPluginFactory::InheritanceChecker<impl>::createInstanceFunction(...))(QWidget*, QObject*, const QVariantList&) [with impl = FileViewSeafilePlugin; KPluginFactory::CreateInstanceFunction = QObject* (*)(QWidget*, QObject*, const QList<QVariant>&); QVariantList = QList<QVariant>]«
/usr/include/KF5/KCoreAddons/kpluginfactory.h:474:73:   von hier erfordert
/usr/include/KF5/KCoreAddons/kpluginfactory.h:520:16: Fehler: ungültiger new-Ausdruck für abstrakte Klasse »FileViewSeafilePlugin«
  520 |         return new impl(p, args);
      |                ^~~~~~~~~~~~~~~~~
In Datei, eingebunden von ../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.cpp:1:
../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.h:9:7: Anmerkung:   denn die folgenden virtuellen Funktionen sind rein innerhalb »FileViewSeafilePlugin«:
    9 | class FileViewSeafilePlugin : public KVersionControlPlugin
      |       ^~~~~~~~~~~~~~~~~~~~~
In Datei, eingebunden von /usr/include/Dolphin/KVersionControlPlugin:1,
                 von ../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.h:4,
                 von ../../dolphin-seafile-plugin-1.2.0/plugin/fileviewseafileplugin.cpp:1:
/usr/include/Dolphin/kversioncontrolplugin.h:186:29: Anmerkung:     »virtual QList<QAction*> KVersionControlPlugin::versionControlActions(const KFileItemList&) const«
  186 |     virtual QList<QAction*> versionControlActions(const KFileItemList& items) const = 0;
      |                             ^~~~~~~~~~~~~~~~~~~~~
/usr/include/Dolphin/kversioncontrolplugin.h:193:29: Anmerkung:     »virtual QList<QAction*> KVersionControlPlugin::outOfVersionControlActions(const KFileItemList&) const«
  193 |     virtual QList<QAction*> outOfVersionControlActions(const KFileItemList& items) const = 0;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:850: fileviewseafileplugin.o] Fehler 1
make[1]: Verzeichnis „/var/tmp/pamac-build/dolphin-seafile-plugin/build/plugin“ wird verlassen
make: *** [Makefile:48: sub-plugin-make_first] Fehler 2

```

--- a/plugin/fileviewseafileplugin.cpp
+++ b/plugin/fileviewseafileplugin.cpp
@@ -74,8 +74,13 @@ KVersionControlPlugin::ItemVersion FileViewSeafilePlugin::itemVersion(const KFil
 		return UnversionedVersion;
 	}
 }
+QList<QAction *> FileViewSeafilePlugin::versionControlActions(const KFileItemList &items) const
+{
+	Q_UNUSED(items)
+	return {};
+}
 
-QList<QAction *> FileViewSeafilePlugin::actions(const KFileItemList &items) const
+QList<QAction *> FileViewSeafilePlugin::outOfVersionControlActions(const KFileItemList &items) const
 {
 //	//test if this it's actually seafile
 //	try {

--- a/plugin/fileviewseafileplugin.h
+++ b/plugin/fileviewseafileplugin.h
@@ -17,7 +17,8 @@ public:
 	bool beginRetrieval(const QString &directory) override;
 	void endRetrieval() override;
 	ItemVersion itemVersion(const KFileItem &item) const override;
-	QList<QAction *> actions(const KFileItemList &items) const override;
+	QList<QAction *> versionControlActions(const KFileItemList &items) const override;
+	QList<QAction *> outOfVersionControlActions(const KFileItemList &items) const override;
 
 private:
 	SeafStatus *_seaf;

